website/static/css/site.css

```diff
@@ -2524,11 +2524,18 @@ div.profile-fullname{
 .file-page #tb-tbody {
     border: none;
 }
-#filesSearch {
-    width: 95%;
+
+#file-navigation .tb-header-row {
+    margin: 0px;
 }
-#toggleIcon {
-    margin-top: 5px;
+
+#file-navigation .tb-table[style] {
+    width: auto !important;
+}
+
+#file-navigation .fa-file {
```

Comment:
This rule could be replaced by adding the `p-l-xs` and `p-r-xs` utility classes to the element.

See [proposed OSF UI guide](http://caneruguz.github.io/osf-style/#layout), under "Layout Helper CSS Classes".